### PR TITLE
Fix #3062: Label added to AdministratorControlsActivity.

### LINKED ISSUE #3062: [A11y] Add label for AdministratorControlsActivity
## Task
Introduce label for `AdministratorControlsActivity` as `Administrator Controls`.

Reference PR: https://github.com/oppia/oppia-android/pull/2750

When you create a PR for this, make sure you add test to check the functionality. Also, suggest working on this PR only if its your first or second PR to Oppia-Android project.

## General Explanation
When `Talkback` is on, whenever we click on any option which leads to new screen the reader will automatically read the `label` associated with that screen first. If in case label in not present then it will read app name i.e. `Oppia`.

### Example

https://user-images.githubusercontent.com/9396084/108906068-78b75080-7646-11eb-943e-212e29fc3e98.mp4

In this above video as soon as we double-tap `Battery`, new screen is opened. 
The reader first outputs `Battery` followed by `Navigate up, Button` and finally `Battery` again. In this the first `Battery` output is the `label` of this new screen which indicates the learner which screen is opened. The second `Battery` is because we selected the title in the screen and it's just reading that text.
So we need `labels` for various screens general which matches the title but if the title is dynamic then we need static label.

### PR DESCRIPTION
**Pull request by jkugsiya**
## Explanation

Fixes #3062 : Added label to AdministratorControlsActivity.
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

Screenshot of successful test:

RunTest

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt:110]: Let's use `internalProfileId` at all places where needed, Thanks for adding this variable

```kt
      createAdministratorControlsActivityIntent(
        profileId = 0
      )
```

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt:129]: please add a screenshot of the espresso test result showing this test case is passing.

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt
Comment (jkugsiya) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt:129]: RunTest

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt
Comment (jkugsiya) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt:110]: Done as suggested here:  https://github.com/oppia/oppia-android/pull/3152/commits/81190dcf51b553fa939f95dc7525f6f1b4dec464

**Comment by anandwana001** [comment]
Follow - https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section

**Comment by jkugsiya** [comment]
> Follow - https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section

I unassigned myself and requested a review.

**Comment by anandwana001** [comment]
@jkugsiya Could you confirm, if you have access to the Assignee section?

**Comment by jkugsiya** [comment]
> @jkugsiya Could you confirm, if you have access to the Assignee section?

No, I don't have access to that! 
image

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt:129]: Could you add this screenshot to the PR description.

Comment (jkugsiya) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt:129]: Added screenshot to the PR description.

File: app/src/main/res/values/strings.xml
Comment (rt4914) [inline app/src/main/res/values/strings.xml]: Rename this to `administrator_controls_title`

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Remove this blank line.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@jkugsiya Thanks.

File: app/src/main/res/values/strings.xml
Comment (jkugsiya) [reply app/src/main/res/values/strings.xml]: Renamed administrator_controls_label to administrator_controls_title: https://github.com/oppia/oppia-android/pull/3152/commits/3ee6278ea4eb85cd5f8ebae54ba6e1125b9bd362

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt
Comment (jkugsiya) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Removed blank line: https://github.com/oppia/oppia-android/pull/3152/commits/3ee6278ea4eb85cd5f8ebae54ba6e1125b9bd362

File: app/src/main/AndroidManifest.xml
Comment (rt4914) [inline app/src/main/AndroidManifest.xml]: This name needs to be updated.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@jkugsiya One nit change.

Also, this PR contains changes related to formatting of the `AndroidManifest` and `strings` file which I think is correct change otherwise other contributors will face this same issue. But I will defer to @BenHenning about this.

File: app/src/main/AndroidManifest.xml
Comment (jkugsiya) [reply app/src/main/AndroidManifest.xml]: Apologies for the mistake. It is corrected in :  https://github.com/oppia/oppia-android/pull/3152/commits/099f9b696aec31a9fc6fc0a3d52808dca6023e2e

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by jkugsiya** [comment]
I've updated the fork's branch and that is reflecting the changes and showing that it is up-to-date. https://github.com/jkugsiya/oppia-android/tree/A11y-AdministratorControlsActivity
updated

But I don't know why the same is not reflecting in the PR.
notUpdated

Can someone help me resolve this problem?

**Comment by anandwana001** [comment]
@jkugsiya As Fetch Upstream is something GitHub has released on the repository directly, but this will not push to your PR. This is only updating your branch on the fork, not pushed into the PR.

**Comment by jkugsiya** [comment]
> @jkugsiya As Fetch Upstream is something GitHub has released on the repository directly, but this will not push to your PR. This is only updating your branch on the fork, not pushed into the PR.

How can I push to the PR then? As all the changes are done and there's nothing to commit more.

**Comment by anandwana001** [comment]
You need to check if your local PR branch is synced with the repository or not. If not then git pull and update the PR, and then you need to push to this PR.

**Comment by jkugsiya** [comment]
> You need to check if your local PR branch is synced with the repository or not. If not then git pull and update the PR, and then you need to push to this PR.

Already done the specified. Still no luck! Shows Everything up-to-date
image

**Comment by jkugsiya** [comment]
> > You need to check if your local PR branch is synced with the repository or not. If not then git pull and update the PR, and then you need to push to this PR.
> 
> Already done the specified. Still no luck! Shows Everything up-to-date
> image

It automatically appeared now, without doing anything. It could have been an issue with GitHub. Thanks @anandwana001 !!
image

**Review by BenHenning — APPROVED** [review]
Thanks @jkugsiya. This LGTM.

**Comment by BenHenning** [comment]
@anandwana001 I think you're requesting changes which is blocking this PR--PTAL.

**Review by anandwana001 — APPROVED** [review]
LGTM

**Pull request merged**
